src/ResourceManager/CognitiveServices/Commands.Management.CognitiveServices/Models/PSCognitiveServicesAccount.cs

```diff
@@ -48,8 +48,8 @@ public PSCognitiveServicesAccount(CognitiveServicesModels.CognitiveServicesAccou
         public string Location { get; private set; }
 
         public Sku Sku { get; private set; }
-        
-        public Kind? AccountType { get; private set; }
+
+        public string AccountType { get; private set; }
```

Comment:
This is needed, just like in SDK we changed it from Enum to String. What should I do here?